### PR TITLE
ME11: ME12: Synced SVD files to Register files. 

### PR DESCRIPTION
**Pull request by sihyung-maxim**
Changes:
- Peripheral SVD files and register files should now match for ME11 and ME12.
- Updated copyright notices.
- Updated max32660.svd and max32662.svd.
- Register files for ME11 and ME12 should now pass linter.
- Resolved SVDConv errors and warnings.

### CODE REVIEW

File: Libraries/CMSIS/Device/Maxim/MAX32660/Include/spi_regs.h [hunk 1]
```diff
@@ -184,31 +202,47 @@ typedef struct {
...
+
+#define MXC_F_SPI_CTRL2_THREE_WIRE_POS                 15 /**< CTRL2_THREE_WIRE Position */
+#define MXC_F_SPI_CTRL2_THREE_WIRE                     ((uint32_t)(0x1UL << MXC_F_SPI_CTRL2_THREE_WIRE_POS)) /**< CTRL2_THREE_WIRE Mask */
+#define MXC_V_SPI_CTRL2_THREE_WIRE_DIS                 ((uint32_t)0x0UL) /**< CTRL2_THREE_WIRE_DIS Value */
+#define MXC_S_SPI_CTRL2_THREE_WIRE_DIS                 (MXC_V_SPI_CTRL2_THREE_WIRE_DIS << MXC_F_SPI_CTRL2_THREE_WIRE_POS) /**< CTRL2_THREE_WIRE_DIS Setting */
+#define MXC_V_SPI_CTRL2_THREE_WIRE_EN                  ((uint32_t)0x1UL) /**< CTRL2_THREE_WIRE_EN Value */
+#define MXC_S_SPI_CTRL2_THREE_WIRE_EN                  (MXC_V_SPI_CTRL2_THREE_WIRE_EN << MXC_F_SPI_CTRL2_THREE_WIRE_POS) /**< CTRL2_THREE_WIRE_EN Setting */
+
+#define MXC_F_SPI_CTRL2_SS_POL_POS                     16 /**< CTRL2_SS_POL Position */
+#define MXC_F_SPI_CTRL2_SS_POL                         ((uint32_t)(0x1UL << MXC_F_SPI_CTRL2_SS_POL_POS)) /**< CTRL2_SS_POL Mask */
```
Comment (Jacob-Scheiffler) [inline Libraries/CMSIS/Device/Maxim/MAX32660/Include/spi_regs.h:241]: Add SS1/2/3_LOW and SS1/2/3_HIGH?

File: Libraries/CMSIS/Device/Maxim/MAX32660/Include/spimss_regs.h [hunk 2]
```diff
@@ -162,29 +194,61 @@ typedef struct {
...
+#define MXC_F_SPIMSS_INT_FL_TXST_POS                   1 /**< INT_FL_TXST Position */
+#define MXC_F_SPIMSS_INT_FL_TXST                       ((uint32_t)(0x1UL << MXC_F_SPIMSS_INT_FL_TXST_POS)) /**< INT_FL_TXST Mask */
+#define MXC_V_SPIMSS_INT_FL_TXST_IDLE                  ((uint32_t)0x0UL) /**< INT_FL_TXST_IDLE Value */
+#define MXC_S_SPIMSS_INT_FL_TXST_IDLE                  (MXC_V_SPIMSS_INT_FL_TXST_IDLE << MXC_F_SPIMSS_INT_FL_TXST_POS) /**< INT_FL_TXST_IDLE Setting */
+#define MXC_V_SPIMSS_INT_FL_TXST_BUSY                  ((uint32_t)0x1UL) /**< INT_FL_TXST_BUSY Value */
+#define MXC_S_SPIMSS_INT_FL_TXST_BUSY                  (MXC_V_SPIMSS_INT_FL_TXST_BUSY << MXC_F_SPIMSS_INT_FL_TXST_POS) /**< INT_FL_TXST_BUSY Setting */
+
+#define MXC_F_SPIMSS_INT_FL_TUND_POS                   2 /**< INT_FL_TUND Position */
+#define MXC_F_SPIMSS_INT_FL_TUND                       ((uint32_t)(0x1UL << MXC_F_SPIMSS_INT_FL_TUND_POS)) /**< INT_FL_TUND Mask */
+#define MXC_V_SPIMSS_INT_FL_TUND_NOEVENT               ((uint32_t)0x0UL) /**< INT_FL_TUND_NOEVENT Value */
```
Comment (Jacob-Scheiffler) [inline Libraries/CMSIS/Device/Maxim/MAX32660/Include/spimss_regs.h:213]: INACTIVE and PENDING?

File: Libraries/CMSIS/Device/Maxim/MAX32660/Include/spimss_regs.h [hunk 3]
```diff
@@ -324,25 +416,45 @@ typedef struct {
...
+#define MXC_S_SPIMSS_I2S_CTRL_I2S_EN_DISABLE           (MXC_V_SPIMSS_I2S_CTRL_I2S_EN_DISABLE << MXC_F_SPIMSS_I2S_CTRL_I2S_EN_POS) /**< I2S_CTRL_I2S_EN_DISABLE Setting */
+#define MXC_V_SPIMSS_I2S_CTRL_I2S_EN_ENABLE            ((uint32_t)0x1UL) /**< I2S_CTRL_I2S_EN_ENABLE Value */
+#define MXC_S_SPIMSS_I2S_CTRL_I2S_EN_ENABLE            (MXC_V_SPIMSS_I2S_CTRL_I2S_EN_ENABLE << MXC_F_SPIMSS_I2S_CTRL_I2S_EN_POS) /**< I2S_CTRL_I2S_EN_ENABLE Setting */
+
+#define MXC_F_SPIMSS_I2S_CTRL_I2S_MUTE_POS             1 /**< I2S_CTRL_I2S_MUTE Position */
+#define MXC_F_SPIMSS_I2S_CTRL_I2S_MUTE                 ((uint32_t)(0x1UL << MXC_F_SPIMSS_I2S_CTRL_I2S_MUTE_POS)) /**< I2S_CTRL_I2S_MUTE Mask */
+#define MXC_V_SPIMSS_I2S_CTRL_I2S_MUTE_NORMAL          ((uint32_t)0x0UL) /**< I2S_CTRL_I2S_MUTE_NORMAL Value */
+#define MXC_S_SPIMSS_I2S_CTRL_I2S_MUTE_NORMAL          (MXC_V_SPIMSS_I2S_CTRL_I2S_MUTE_NORMAL << MXC_F_SPIMSS_I2S_CTRL_I2S_MUTE_POS) /**< I2S_CTRL_I2S_MUTE_NORMAL Setting */
+#define MXC_V_SPIMSS_I2S_CTRL_I2S_MUTE_REPLACED        ((uint32_t)0x1UL) /**< I2S_CTRL_I2S_MUTE_REPLACED Value */
```
Comment (Jacob-Scheiffler) [inline Libraries/CMSIS/Device/Maxim/MAX32660/Include/spimss_regs.h:430]: I2s_MUTE_MUTE instead? Or Omit?

**Review by Jacob-Scheiffler — CHANGES_REQUESTED** [review]

File: Libraries/CMSIS/Device/Maxim/MAX32660/Include/gpio_regs.h [hunk 4]
```diff
@@ -132,73 +132,73 @@ typedef struct {
...
+#define MXC_R_GPIO_INT_DUAL_EDGE           ((uint32_t)0x0000005CUL) /**< Offset from GPIO Base Address: <tt> 0x005C</tt> */
+#define MXC_R_GPIO_PAD_CFG1                ((uint32_t)0x00000060UL) /**< Offset from GPIO Base Address: <tt> 0x0060</tt> */
+#define MXC_R_GPIO_PAD_CFG2                ((uint32_t)0x00000064UL) /**< Offset from GPIO Base Address: <tt> 0x0064</tt> */
+#define MXC_R_GPIO_EN1                     ((uint32_t)0x00000068UL) /**< Offset from GPIO Base Address: <tt> 0x0068</tt> */
+#define MXC_R_GPIO_EN1_SET                 ((uint32_t)0x0000006CUL) /**< Offset from GPIO Base Address: <tt> 0x006C</tt> */
+#define MXC_R_GPIO_EN1_CLR                 ((uint32_t)0x00000070UL) /**< Offset from GPIO Base Address: <tt> 0x0070</tt> */
+#define MXC_R_GPIO_EN2                     ((uint32_t)0x00000074UL) /**< Offset from GPIO Base Address: <tt> 0x0074</tt> */
+#define MXC_R_GPIO_EN2_SET                 ((uint32_t)0x00000078UL) /**< Offset from GPIO Base Address: <tt> 0x0078</tt> */
+#define MXC_R_GPIO_EN2_CLR                 ((uint32_t)0x0000007CUL) /**< Offset from GPIO Base Address: <tt> 0x007C</tt> */
```
[the remark this answers is not in the document]
Comment (sihyung-maxim) [reply Libraries/CMSIS/Device/Maxim/MAX32660/Include/gpio_regs.h:167]: The ME11 doesn't support the addition of the EN3 registers. The ME12 was the first part to introduce the new set of EN3 registers.

The ME11 UG uses IS and SR names.

**Comment by sihyung-maxim** [comment]
I removed the enum values for fields that are 1 bit wide (as they should be).

**Review by lorne-maxim — APPROVED** [review]

**Review by Jacob-Scheiffler — APPROVED** [review]

**Pull request merged**